volume-group-operator/Makefile

```diff
@@ -19,7 +19,6 @@ help: ## Display this help.
 manifests: controller-gen kustomize## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=volume-group-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	hack/update_labels_in_crd.sh
-	hack/update_labels_in_rbac.sh
```

Comment:
why did you remove this line?

Reply:
I didnt, can it be one of the scripts?

Reply:
which scripts? and GitHub shows that the line removed

Reply:
done